README.md

```diff
@@ -10,10 +10,17 @@ Steps to run a local copy of SpartaHackAPI
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3`  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`export WORKON_HOME=~/.virtualenvs`  
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`source /usr/local/bin/virtualenvwrapper.sh`  
-2. Clone the API and cd into it
-3. Create a virtualenv using `mkvirtualenv` using the virtualenvwrapper
-4. Switch the python interpreter to the virtualenv using `workon virtualenv_name` 
-5. Install the dependencies using `pip install -r requirements.txt`
+2. Clone the API and cd into it  
+3. Create a virtualenv using `mkvirtualenv` using the virtualenvwrapper  
+4. Switch the python interpreter to the virtualenv using `workon virtualenv_name`  
+5. Install the dependencies using `pip install -r requirements.txt`  
+6. Install RabbitMQ server using `sudo apt install rabbitmq-server`  
+7. With RabbitMQ server running (it will after first install otherwise run it using `sudo rabbitmq-server`)  
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`sudo rabbitmqctl add_user dev mypassword`  
```

Comment:
This looks pretty bad. Is there a better way to hack it. or list within list?

Reply:
Let me see.